index.d.ts

```diff
@@ -0,0 +1,33 @@
+declare module 'joi-password-complexity'{
+  import { ValidationError } from '@hapi/joi';
```

Comment:
We need to support the new joi also.
`const joi = softRequire('joi') || softRequire('@hapi/joi');`
How can we go about that?

Reply:
Hummmm I Typescript does not seem to be able to type it's way through with getJoi. 
I could re-type the ErrorValidation so we don't need imports

Reply:
Like so 

```javascript
  interface ValidationErrorItem {
    message: string;
    type: string;
    path: string[];
    options?: ValidationOptions;
    context?: Context;
  }

  interface ValidationError extends Error {
    details: ValidationErrorItem[];
    annotate(): string;
    _object: any;
  }
```

Reply:
That should work